packages/volto-slate/src/editor/render.jsx

```diff
@@ -152,23 +158,57 @@ export const serializeNodesToHtml = (nodes) =>
   renderToStaticMarkup(serializeNodes(nodes));
 
 export const renderLinkElement = (tagName) => {
-  function LinkElement({ attributes, children, mode = 'edit' }) {
+  function LinkElement({
+    attributes,
+    children,
+    mode = 'edit',
+    className = null,
+  }) {
+    const { slate = {} } = config.settings;
     const Tag = tagName;
     const slug = attributes.id || '';
+    const location = useLocation();
+    const appPathname = addAppURL(location.pathname);
+    // eslint-disable-next-line no-unused-vars
+    const [copied, copy, setCopied] = useClipboard(
+      appPathname.concat(`#${slug}`),
+    );
+    const intl = useIntl();
 
-    return (
-      <Tag {...attributes}>
+    return slate.useLinkedHeadings === false ? (
+      <Tag {...attributes} className={className}>
+        {children}
+      </Tag>
+    ) : (
+      <Tag {...attributes} className={className}>
+        {children}
         {mode === 'view' && slug && (
-          <a class="anchor" aria-hidden="true" href={`#${slug}`}>
-            <svg
-              {...linkSVG.attributes}
-              dangerouslySetInnerHTML={{ __html: linkSVG.content }}
-              width="2em"
-              height={null}
-            ></svg>
-          </a>
+          <span className="anchor-wrapper">
```

Comment:
Is the wrapper really needed? It looks to me like it would be possible to put the margin and vertical-align on the link itself if it uses `display: inline-block` instead of `display: contents`. It would be nice to keep the markup simple and semantic, especially since this is part of the public UI.

Reply:
Removed in [c33f634](https://github.com/plone/volto/pull/4287/commits/c33f634e32bcd7eb56ae80d36dd5258f2aeb101e)